### PR TITLE
Provides conan file to build artifacts

### PR DESCRIPTION
**Pull request by deekshith-elear**
Adds conan recipe file to this repo

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+class SqlitelibConan(ConanFile):
+    name = "sqlite"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of Sqlitelib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
```
Comment (manavkumarm) [inline conanfile.py]: Fix spacing as per httpserver feedback

**Review by manavkumarm — CHANGES_REQUESTED** [review]

File: conanfile.py (hunk 1 above)
Comment (deekshith-elear) [reply conanfile.py]: handled

File: conanfile.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class SqlitelibConan(ConanFile):
+    name = "sqlite"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of Sqlitelib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = { "shared": [True, False] }
```
Comment (manavkumarm) [inline conanfile.py]: Pending style fix: `[ True, False ]` - space after [, space before ]

**Review by manavkumarm — APPROVED** [review]

File: conanfile.py (hunk 2 above)
Comment (deekshith-elear) [reply conanfile.py]: handled

**Pull request merged**